File: c/3052/3052.c
#include <stdio.h>

int main(){

    int arr[42]={0};
    int n;
    for(int i=0; i<10;i++){
        scanf("%d",&n);
        arr[n%42]=1;
    }

    int sum=0;
    for (int i=0;i<42;i++){
        if (arr[i]==1) sum+=1;
    }

    printf("%d\n",sum);


    return 0;
}

File: python/3052/3052.py
import sys
input = sys.stdin.readline

arr=[0]*42

for i in range(10):
    n=int(input())
    if arr[n%42]==0:
        arr[n%42]+=1

sum=0        

for _ in arr:
    if _ == 1:
        sum+=1
print(sum)